src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.Storage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+
+    @Test
+    void getReport_validStorage_ok() {
```

Comment:
Let's add more test cases (for empty storage for example)
Don't forget clear storage after testing. Check all test classes!!!

Reply:
changes have been made